models/check_model.py

```diff
@@ -1,22 +1,50 @@
 import discord
+
 from models.env_service_model import EnvService
 from typing import Callable
 
-
-ALLOWED_ROLES = EnvService.get_allowed_roles()
+ADMIN_ROLES = EnvService.get_admin_roles()
```

Comment:
I will probably need you to change the way that the roles are retrieved soon.. unfortunately the current EnvService is riddled with bugs as per #41 and I'm going to be implementing a new way shortly, I apologize for the inconvenience, and I'll ping you when I've rolled out the new changes for that

Reply:
Nvm on this, I think .env stuff was fixed without many major changes, but I'd still like to wait for a confirmed resolution on #41 before we merge this